### DISCUSSION TITLE
override enabledManagers in config presets

### DISCUSSION THREAD

**Discussion by hisabimbola** [renovatebot/renovate#10277] [Request Help]
Hi all,

I have a base preset which has some manager enabled in the "enabledManagers" property. 
When an application uses the preset and wants to enable additional managers, the managers in the presets take precedence and only that is enabled, this makes it impossible for applications to enable additional managers or even override what is defined in the preset

I wonder if there is any config like "addManagers" and this will add more managers to the list of already enabled managers or does anybody have a workaround to achieve this

**Comment by rarkins** [comment]
I think you should be able to use `enabledManagers` to override `enabledManagers`. The list in the downstream repo should take precedence of the list in the preset.
